examples/async_fl/async_aggregator.py

```diff
@@ -28,7 +28,9 @@ def __init__(self, args):
         self.round_stamp = [0]
         self.client_model_version = {}
         self.virtual_client_clock = {}
-        self.round_lock = threading.Lock()
+        self.weight_tensor_type = {}
+        # We need to keep the test model for specific round to avoid async mismatch
+        self.test_model = None
 
     def tictak_client_tasks(self, sampled_clients, num_clients_to_collect):
```

Comment:
In this async aggregator implementation, is there a notion of concurrency? In the PAPAYA paper, concurrency is a hyper-parameter in addition to the buffer size. 
<img width="435" alt="image" src="https://user-images.githubusercontent.com/12105532/184684060-bdc5388a-5ee1-4df7-9617-bb46debc76a6.png">

Reply:
That's a good point and sorry for not combining Papaya's design. We should do 2 more things
1) add the concurrency hyper-parameter
2) Decide the model version of every participants in `create_client_task` by 
```
if first_round:
    model_id = 0
else:
    model_id = self.find_latest_model(prev_client_end_time) // if assume there are always devices awaiting
```
instead of https://github.com/SymbioticLab/FedScale/blob/51cc4a1e0ab553cd79ecb59af211008788f1af39/examples/async_fl/async_aggregator.py#L213-L214

I will talk to Fan to work on this asap